polyfill/lib/ecmascript.mjs

```diff
@@ -1236,7 +1234,18 @@ export const ES = ObjectAssign({}, ES2020, {
       }
 
       calendar = ES.GetOptionalTemporalCalendar(item);
-      const fieldNames = ES.CalendarFields(calendar, ['day', 'month', 'monthCode', 'year']);
+      const fieldNames = ES.CalendarFields(calendar, [
```

Comment:
Oh dear, this bug must've been here a while. Will add a test262 work item for this and the one below in ToTemporalZonedDateTime, good catch.